tests/test.js

```diff
@@ -111,7 +115,41 @@ describe('functionality', function() {
     });
 
     it('should dispatch event', function() {
-      vote.should.be.equal('3')
+      vote.should.be.equal('3');
+    });
+
+    it('should delete active class', function() {
+      star.classList.contains(testOptions.activeClass).should.be.false;
+    });
+  });
+
+  context('functions and events', function() {
```

Comment:
I guess we could call this context better, e.g. 
```suggestion
  context('mouse leave', function() {
```